src/machine/CoffeeMachine.java

```diff
@@ -0,0 +1,80 @@
+package machine;
+
+class CoffeeMachine {
+
+    private final Supplies supplies;
+
+    // For our purposes it is sufficient that the price has int type
+    private int money;
```

Comment:
Let's rename it to `dollars`? 

Also would be nice if the `For our purposes it is sufficient that the price has int type` comment was the same here and in the `Coffee` enum